templates/docs/examples/patterns/breadcrumbs.html

```diff
@@ -4,12 +4,15 @@
 {% block standalone_css %}patterns_breadcrumbs{% endblock %}
 
 {% block content %}
-<ul class="p-breadcrumbs">
-  <li class="p-breadcrumbs__item">Overview</li>
-  <li class="p-breadcrumbs__item"><a href="#">Features</a></li>
-  <li class="p-breadcrumbs__item"><a href="#">Managed</a></li>
-  <li class="p-breadcrumbs__item"><a href="#">Install</a></li>
-  <li class="p-breadcrumbs__item"><a href="#">Partners</a></li>
-  <li class="p-breadcrumbs__item"><a href="#">Docs</a></li>
-</ul>
+<nav class="p-breadcrumbs" aria-label="Breadcrumb">
+  <ol class="p-breadcrumbs__items">
+      <li class="p-breadcrumbs__item"><a href="#">Overview</a></li>
+      <li class="p-breadcrumbs__item"><a href="#">Features</a></li>
+      <li class="p-breadcrumbs__item"><a href="#">Managed</a></li>
+      <li class="p-breadcrumbs__item"><a href="#">Install</a></li>
+      <li class="p-breadcrumbs__item"><a href="#">Partners</a></li>
+      <li class="p-breadcrumbs__item">Docs</li>
```

Comment:
Should we add `aria-current="page"` while we are at it?

Reply:
It would require wrapping in an `anchor`, which seems pointless. The spec doesn't recommend using it when no link is present.

Reply:
Ok, I see [bootstrap is doing in on `li` ](https://getbootstrap.com/docs/4.0/components/breadcrumb/#overview) but if spec doesn't recommend maybe we shouldn't.